Bigtable/src/DataClient.php

```diff
@@ -266,6 +279,17 @@ public function readRows(array $options = [])
                 ]
             );
         }
+        if ($filter !== null) {
+            if (!$filter instanceof FilterInterface) {
+                throw new \InvalidArgumentException(
+                    sprintf(
+                        'Expected filter to be of type FilterInterface, instead got \'%s\'.',
```

Comment:
WDYT about the following?

```php
sprintf(
    'Expected filter to be of type \'%s\', instead got \'%s\'.',
    FilterInterface::class,
    gettype($filter)
)
```

The benefits being users can see the fully qualified name space, and if we end up changing the namespace at all we just need to worry about changing the import at the top of the file.